integration/test_flow_integration.py

```diff
@@ -320,7 +343,7 @@ def test_join_by_key(setup_kv_teardown_test):
 
 
 def test_join_by_key_specific_attributes(setup_kv_teardown_test):
-    table = Table(setup_kv_teardown_test, V3ioDriver())
+    table = Table(setup_kv_teardown_test["table_name"], get_driver(setup_kv_teardown_test))
```

Comment:
We use single quotes in storey, except for `"""` and when single quotes need to appear in the actual string.
```suggestion
    table = Table(setup_kv_teardown_test['table_name'], get_driver(setup_kv_teardown_test))
```

Reply:
removed by introducing TestContext getters.